egs/ami/s5b/RESULTS_ihm

```diff
@@ -84,6 +84,10 @@
 %WER 20.8 | 13098 94489 | 82.0 10.0 8.0 2.8 20.8 53.2 | -0.096 | exp/ihm/chain_cleaned/tdnn_lstm1i_sp_bi_ld5/decode_dev/ascore_11/dev_hires.ctm.filt.sys
 %WER 20.7 | 12643 89980 | 81.7 11.5 6.8 2.5 20.7 51.8 | 0.015 | exp/ihm/chain_cleaned/tdnn_lstm1i_sp_bi_ld5/decode_eval/ascore_11/eval_hires.ctm.filt.sys
 
+# local/chain/tuning/run_tdnn_lstm_1m.sh --mic ihm  --train-set train_cleaned  --gmm tri3_cleaned
```

Comment:
Does 1i already account for all the new changes like fast-lstm component. If not please specify what all things changed between 1i and 1m.